Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM dlanguage/dmd
+LABEL "Version"="1.0"
+
+COPY . /opt/psi
+
+RUN apt -qq update && apt -qq install -y wget unzip
+RUN cd /opt/psi && ./dependencies.sh && ./build.sh && mkdir bin && mv psi ./bin
```

Comment:
This should use dependencies-release.sh and build-release.sh instead (otherwise people using docker will get fast builds but bad performance).